cmd/deployctl/cmd/build.go

```diff
@@ -167,14 +168,15 @@ func CollectCmdRun(cmd *cobra.Command, args []string) {
 	}
 
 	if minimalConfig {
-		noCACertificates = true
 		noDefaults = true
 		noInterfaceDefaults = true
 		normalizeInterfaces = true
 		normalizeMTU = true
 		normalizeConsole = true
 		noServiceParams = true
 	}
+	noDRBDLinkUtilization = true
+	noCACertificates = true
 
```

Comment:
Would expect two additional arguments to allow the generation of these two types of values

Reply:
I have added 3 arguments for the attributes
ssl_ca: "--allow-ca-certficates"
DRBD link utilization: "--allow-drbd"
All System.spec.storage.filesystems attributes: "--allow-all-fs"